vignettes/imputation.Rmd

```diff
@@ -121,8 +127,9 @@ desirable. The `{admiral}` imputation functions provide the `min_dates` and
 `max_dates` parameter to specify those dates. For example:
 
 ```{r}
-impute_dtc(
+impute_dtc_dtm(
   "2019-02",
+  highest_imputation = "M",
   date_imputation = "last",
   time_imputation = "last",
   max_dates = list(ymd("2019-01-14"), ymd("2019-02-25"))
```

Comment:
i would add to this section 2 examples where `highest_imputation="Y"` used. 1 with `"first"` & `min_dates`, 1 with `"last"` & `max_dates`

Reply:
Added